shared/chat/inbox/container.js

```diff
@@ -11,16 +12,28 @@ const getInbox = (state: TypedState) => state.chat.get('inbox')
 const getSupersededByState = (state: TypedState) => state.chat.get('supersededByState')
 const getAlwaysShow = (state: TypedState) => state.chat.get('alwaysShow')
 const getPending = (state: TypedState) => state.chat.get('pendingConversations')
+const getFilter = (state: TypedState) => state.chat.get('inboxFilter')
 
 const createImmutableEqualSelector = createSelectorCreator(defaultMemoize, I.is)
 
+const passesFilter = (i: Constants.InboxState, filter: I.List<string>): boolean => {
+  if (!filter.count()) {
+    return true
+  }
+
+  const participants = i.get('participants')
+  return filter.every(f => participants.indexOf(f) !== -1)
```

Comment:
You can replace this and the line above with: `filter.isSubset(i.get('participants'))`